src/main/java/org/broadinstitute/consent/http/enumeration/VoteType.java

```diff
@@ -4,7 +4,7 @@
 
 public enum VoteType {
 
-  DAC("DAC"), FINAL("FINAL"), DACBOTAPPROVE("DACBOTAPPROVE"), AGREEMENT("AGREEMENT"), CHAIRPERSON(Resource.CHAIRPERSON);
+  DAC("DAC"), FINAL("FINAL"), RADARAPPROVE("RADARAPPROVE"), AGREEMENT("AGREEMENT"), CHAIRPERSON(Resource.CHAIRPERSON);
```

Comment:
Minor suggestion, can we underscore-separate this like we do for [ElectionType](https://github.com/DataBiosphere/consent/blob/a98a903efab8cb085715af21330980c7d9deef19/src/main/java/org/broadinstitute/consent/http/enumeration/ElectionType.java#L8)s?
```suggestion
  DAC("DAC"), FINAL("FINAL"), RADAR_APPROVE("RADAR_APPROVE"), AGREEMENT("AGREEMENT"), CHAIRPERSON(Resource.CHAIRPERSON);
```